### ISSUE TITLE
508-defect-3 [COGNITION]: Ch31 home, lists should be announced as lists in VoiceOver

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22831]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a VoiceOver user, I expect to hear lists announced as lists

## Details

VoiceOver will not announce lists as lists if there is no bullet styling. This can be easily fixed by adding in `role="list"` to the `ul`

## Acceptance Criteria

- [ ] VoiceOver announces all lists as lists

## Environment

Mac, Safari, VoiceOver

## Steps to Recreate

1. Enter `https://staging.va.gov/careers-employment/vocational-rehabilitation/` in browser
2. Start VoiceOver
3. Using VO keys, navigate to the headings list
4. Confirm they are not announced as lists

## Proposed Solution

Simply add `role="list"` to the `ul`

## WCAG or Vendor Guidance (optional)

* [Fixing lists by Scott o Hara](https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html)

**Comment by ohgreatkate** [comment]
completed.
